### PR TITLE
Reset button appears after changing hydrograph

### PR DESCRIPTION
**Pull request by skaymen**

### CODE REVIEW

File: floodviz/static/css/hydrograph.css [hunk 1]
```diff
@@ -102,3 +102,10 @@
     stroke-linejoin: round;
     stroke-miterlimit: 10.00;
 }
+
+.btn {
```
Comment (jameeters) [inline floodviz/static/css/hydrograph.css]: It might be better to style it by ID rather than class.

File: floodviz/templates/index.html [hunk 2]
```diff
@@ -59,7 +59,7 @@
 				</div>
 			</div>
 			<div class="col-sm-6 col-centered">
-				<div id="hydrograph"></div>
+				<div id="hydrograph"><input class="btn btn-primary pull-right" id="button" type="reset" value="Reset"></div>
```
Comment (jameeters) [inline floodviz/templates/index.html]: 'reset-button' would probably be a better id.

File: floodviz/static/js/hydrograph.js [hunk 3]
```diff
@@ -228,6 +230,16 @@ var hydromodule = function (options) {
 			.style("font-size", "14px")
 			.text("Discharge (cubic feet per second)");
 
+		var button = d3.select(".btn");
+		button.on('click', function() {reset_hydrograph();});
+
+		if (default_display_ids.toString() === options.display_ids.toString()) {
```
Comment (ede0m) [inline floodviz/static/js/hydrograph.js]: why do we have to compare by string here?

File: floodviz/templates/index.html (hunk 2 above)
Comment (skaymen) [inline floodviz/templates/index.html]: Fixed.

File: floodviz/static/css/hydrograph.css (hunk 1 above)
Comment (skaymen) [inline floodviz/static/css/hydrograph.css]: Fixed.

File: floodviz/static/js/hydrograph.js (hunk 3 above)
Comment (mbucknell) [inline floodviz/static/js/hydrograph.js]: This comparision makes me nervous. You are relying on the ids being in the same order. Okay for now, but it seems to me we are doing far too much in this update procedure. It is over 150 lines. If we had more time, I would look at refactoring the hydrograph module.

**Pull request merged**